### ISSUE TITLE
Vue - isAuthenticated returns true temporarily after browser close

### ISSUE THREAD

**Issue by rsflaherty** [okta/okta-oidc-js#267]
I am seeing an issue consistently when I perform the following steps:
1. access my site and login
2. close my browser
3. reopen the browser and access a protected url on the site.  

The isAuthenticated() method seems to return true temporarily when this happens which allows the browser to continue to the protected url.  Oddly enough, navigating to a second protected route will trigger the login.

At the same time this happens, I see an error in the console that says "The client specified not to prompt, but the user is not logged in." Similar to issue #266 but wasn't sure if it was exactly the same since I am also seeing the isAuthenticated return true.

Need a way to either trap the error message or get the isAuthenticated method to return false.

Have the following setup in my router to check that the user is logged in when requiresAuth is set in the meta of the route.  I am using the Okta Widget on my login page

```
const onAuthRequired = async (from, to, next) => {
  if (from.matched.some(record => record.meta.requiresAuth) && !(await Vue.prototype.$auth.isAuthenticated())) { 
    // Navigate to custom login page
    next({ path: '/login' })
  } else {
    next()
  }
}

router.beforeEach(onAuthRequired)
```

**Comment by manueltanzi-okta** [comment]
Hi @rsflaherty , thank you for submitting your issue!

I will try to reproduce it and let you know what I find out!

**Comment by rsflaherty** [comment]
Great, thanks.  Please let me know if you need any more information from me.

**Comment by manueltanzi-okta** [comment]
Hi @rsflaherty,

Good news! I was able to reproduce your bug. It is a bug of okta-auth-js 2.0.0 that was fixed in okta-auth-js 2.0.1.
The okta-vue library is currently using okta-auth-js 2.0.0, so I sent out a [PR](https://github.com/okta/okta-oidc-js/pull/287) to update it to 2.0.1, so that you will not get that error anymore.

However, after the fix, the default behavior will be that when the user close and reopen the browser, it will be still authenticated and it will stay authenticated even if you go in another protected page, since the tokens are stored in localStorage which persist even if you close the browser.

If this is not the behavior you want, I opened another [issue](https://github.com/okta/okta-oidc-js/issues/285) for allowing the developer to choose the storage type in which store the tokens, so, after we will add this feature, you will be able to select sessionStorage instead of localStorage, so that the user will lose authentication after he closes the browser.

**Comment by rsflaherty** [comment]
Thanks!  It helps for now to know that it will be intended behavior.  I think having the feature to use sessionStorage instead would be a good thing as we may have compliance items that require us to force the user to log in again if they close the browser.  Haven't crossed that bridge yet on my project but anticipating for later.

**Comment by manueltanzi-okta** [comment]
Hi @rsflaherty,

We published okta-vue v1.0.3 which include the fix I explained in my previous comment. You can now try the new version on your application!

Will resolve this issue since new version includes the fix to this bug, thank you for reporting it!
